xbmc/cores/AudioEngine/AESinkFactory.h

```diff
@@ -48,7 +48,7 @@ class CAESinkFactory
 
   static void ParseDevice(std::string &device, std::string &driver);
   static IAESink *Create(std::string &device, AEAudioFormat &desiredFormat);
-  static void EnumerateEx(std::vector<AESinkInfo> &list, bool force);
+  static void EnumerateEx(std::vector<AESinkInfo> &list, bool force, std::string driver);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types